contents/data/sql/SQL.md

```diff
@@ -0,0 +1,145 @@
+<frontmatter>
+  title: Introduction to SQL
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Introduction to SQL
+
+**Author(s): [Amrut Prabhu](https://github.com/amrut-prabhu)**
+
+Reviewers: [Ronak Lakhotia](https://github.com/RonakLakhotia), [Rahul Rajesh](https://github.com/rrtheonlyone)
+
+## What is SQL?
+
+Databases are integral to any commercial software application, whether it is a web app, desktop app or otherwise.
+As the size of your application grows, you need to look into dedicated database software packages (like SQL software) to store your data. It will not be enough to simply rely on in-memory solutions or a simple text file to store the data. Many companies like Facebook, Google, Adobe, Paytm and Zappos use <tooltip content="Relational DataBase Management System">RDBMS</tooltip> software to fulfil this need.
```

Comment:
`As the size of your application grows, you need to look into dedicated database software packages`: this doesn't sound right either. It seems to assume people will start with an in-memory database or a text file and migrate to a database later.  I don't think that happens too often.